api/v2/performanceprofile_validation_test.go

```diff
@@ -28,12 +28,23 @@ const (
 
 	//MachineConfigLabelKey defines the MachineConfig label key of the test profile
 	MachineConfigLabelKey = "mcKey"
-	//MachineConfigLabelValue defines the MachineConfig label vlue of the test profile
+	//MachineConfigLabelValue defines the MachineConfig label value of the test profile
 	MachineConfigLabelValue = "mcValue"
 	//MachineConfigPoolLabelKey defines the MachineConfigPool label key of the test profile
 	MachineConfigPoolLabelKey = "mcpKey"
 	//MachineConfigPoolLabelValue defines the MachineConfigPool label value of the test profile
 	MachineConfigPoolLabelValue = "mcpValue"
+
+	//NetDeviceName defines a net device name for the test profile
+	NetDeviceName = "enp0s4"
+	//NetDeviceVendorID defines a net device vendor ID for the test profile
+	NetDeviceVendorID = "0x1af4"
+	//NetDeviceModelID defines a net device model ID for the test profile
+	NetDeviceModelID = "0x1000"
+	//NetDeviceMAC defines a net device PCI path for the test profile
+	NetDevicePCIpath = "pci-0000:00:04.0"
```

Comment:
This seems like a plain PCI address in DOMAIN-BUS-DEVICE-FUNCTIOn format. Is it so? If so why do we call it path?

Reply:
following the udev convention here, for example:
udevadm info -p /sys/class/net/ens4
P: /devices/pci0000:00/0000:00:04.0/virtio1/net/ens4
...
E: ID_PATH=pci-0000:00:04.0

Reply:
But maybe I should remove PCI and just call it interfacePath

Reply:
But isn't this "path" in the form "pci-$PCI_ADDRESS" where PCI_ADDRESS="0000:00:04.0" (DOMAIN:BUS:DEVICE.FUNCTION) ?
meaning: shouldn't it be `NetDevicePCIAddress` ?

Reply:
@fromanirh I wonder if this could be usb path as well (or some other bus). Probably not on a server, but..

Reply:
fair point, but we have `PCI` hardcoded in the name of the field already (`NetDevicePCIpath`).
My biggest gripe is that "PCI path" is a quite hard concept to parse and seems very udev-specific. We should not expose udev-specific details in our API, especially considering that a plain simple PCI address seems to be equivalent and much simpler concept.

Reply:
Please see my comment above, just use domain:bus:device:function, please don;t use something that is not universally accepted. The same reservations about using PCI addresses

Reply:
Given all the above - since we will probably need a way to translate the PCI address to something udev understands and the risk of PCI addresses not being guaranteed I wonder how badly to we want this field.
@MarSik  thoughts here ?

Reply:
@marcel-apf could you please link docs (kernel or whatever) that explain this behaviour?